### PR TITLE
border-radius as prop for View #117

### PR DESCRIPTION
**Pull request by hayanisaid**
- Add BorderRadius as prop to the View.
- Updated the way the View is created
   - Instead of using `ctx.rect` to draw a rectangle, `moveTo()`, `lineTo()`, `quadraticCurveTo()` are used to allow a dynamic BorderRadius.
   - I couldn't' find a way around `rect` to generate a borderRadius based on a dynamic value, please let me know if this is aright way for this issue.

### CODE REVIEW

File: packages/docs/components-listview.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  return (
+    <View style={styles.item}>
+      <Text style={styles.title}>{title}</Text>
+    </View>
+  );
+}
+
+export default function App() {
+  return (
+      <SectionList
```
Comment (raphamorim) [inline packages/docs/components-listview.md:41]: I don't think we have `ListView` working that way yet and should be `ListView` instead of `SectionList`

**Comment by raphamorim** [comment]
Great work @hayanisaid 🥂 
Would you mind to remove the `SectionList` addition from the docs? Because we still don't have that yet and neither `ListView` component is ready

File: packages/docs/components-listview.md (hunk 1 above)
Comment (hayanisaid) [reply packages/docs/components-listview.md]: Fixed!

**Comment by raphamorim** [comment]
Perfect! Thanks @hayanisaid. Closes #117

**Pull request merged**